src/toil/job.py

```diff
@@ -1057,7 +1058,7 @@ def _serialiseJob(self, jobStore, jobsToJobGraphs, rootJobGraph):
         # for the mechanism which unpickles the job and executes the Job.run
         # method.
         with jobStore.writeFileStream(rootJobGraph.jobStoreID) as (fileHandle, fileStoreID):
-            cPickle.dump(self, fileHandle, cPickle.HIGHEST_PROTOCOL)
+            pickle.dump(self, fileHandle)
```

Comment:
cPickle doesn't correctly dump the CWL objects, even without HIGHEST_PROTOCOL. Sorry it's frustrating, it complains about `Can't pickle <type 'instancemethod'>`